### PR TITLE
Tidy up a couple more one-line and unused styles/types

### PR DESCRIPTION
**Pull request by alexwlchan**
For https://github.com/wellcomecollection/wellcomecollection.org/issues/8420

### CODE REVIEW

File: common/views/components/ToolbarSegmentedControl/ToolbarSegmentedControl.tsx [hunk 1]
```diff
@@ -5,10 +5,13 @@ import { font } from '@weco/common/utils/classnames';
 import Space from '../styled/Space';
 import { IconSvg } from '@weco/common/icons';
 
-const List = styled.ul.attrs({
-  className:
-    'no-margin no-padding plain-list flex-inline flex--v-center flex--h-center',
-})`
+const List = styled.ul`
+  margin: 0 !important;
```
Comment (rcantin-w) [inline common/views/components/ToolbarSegmentedControl/ToolbarSegmentedControl.tsx:9]: Is the `!important` necessary? (I know it was in the class, I just wonder if we could look at it case by case?)

Comment (rcantin-w) [reply common/views/components/ToolbarSegmentedControl/ToolbarSegmentedControl.tsx:9]: + same for the two instances below

**Review by davidpmccormick — COMMENTED** [review]
Is the ultimate goal to get rid of utility classes altogether?

**Comment by rcantin-w** [comment]
> Is the ultimate goal to get rid of utility classes altogether?

Let's discuss in our meeting tomorrow, I think they can still be useful, but if we're talking about focusing on styled-components, reducing the amount of style by class kind of fits with that. Team decision, though!

**Comment by alexwlchan** [comment]
Closing as outdated; I've not had time for this and I'm not going to get to it soon.

**Pull request closed without merging**